lib/PuppeteerSharp/Launcher.cs

```diff
@@ -40,21 +43,28 @@ public class Launcher
 
         #region Private members
         private Process _chromeProcess;
-        private string _temporaryUserDataDir = null;
-        private Connection _connection = null;
-        private Timer _timer = null;
+        private string _temporaryUserDataDir;
+        private Connection _connection;
+        private Timer _timer;
         private LaunchOptions _options;
         private TaskCompletionSource<bool> _waitForChromeToClose;
-        private static int _processCount = 0;
+        private static int _processCount;
```

Comment:
maybe move this field to be after/before all of the instance fields since it's statuc